rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2022-05-24
+nightly-2022-05-25
```

Comment:
If a toolchain update is really needed, send it as a standalone PR.

Reply:
thx [825eaec](https://github.com/singularity-data/risingwave/pull/2822/commits/825eaeca58052deb9b9ade74595ba551b5c0ea2e)